java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1700,8 +1687,8 @@ public Table repeat(ColumnView counts) {
    * @return the new Table.
    * @throws CudfException on any error.
    */
-  public Table repeat(ColumnView counts, boolean checkCount) {
-    return new Table(repeatColumnCount(this.nativeHandle, counts.getNativeView(), checkCount));
+  public Table repeat(ColumnView counts) {
```

Comment:
You forgot to remove Javadoc for `checkCount`.

Reply:
Ah I missed that thanks.